app/assets/stylesheets/components/_separator.sass

```diff
@@ -0,0 +1,4 @@
+$Separator-color-horizontal: Theme-color(line, separator-content)
+
+.Separator-horizontal
```

Comment:
it is the name?
I made it generic `.Separator` and have created a descendant component `Separator-horizontal`.
Then i have removed the `.Separator` because it didn't had properties. 
In this case if we have another kind of separator wi may put it here.